src/entity/SquadPostModeration.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToOne,
+  PrimaryColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { Post } from './posts';
+import { Source } from './Source';
+import { User } from './user';
+
+export enum SquadPostModerationStatus {
+  Approved = 'approved',
+  Rejected = 'rejected',
+  Pending = 'pending',
+}
+
+@Entity()
+export class SquadPostModeration {
```

Comment:
I see its named differently in  diferent places of RFC but I think it should be named `SourcePostModeration` because squads are just sources under the hood so this can potentially be applied to any post (not just squad posts). 

That said its more of a naming thing so lowest prio.